backend/frontend/views.py

```diff
@@ -50,13 +64,111 @@
 ]
 
 
-from django.contrib.auth.decorators import user_passes_test
-
-
 def is_super(user):
     return user.is_superuser
 
 
+def user_login(request):
+    form = CustomAuthenticationForm()
+    context = {"form": form}
+
+    if request.method == "POST":
+        form = CustomAuthenticationForm(request=request, data=request.POST)
+        context.update({"form": form})
+
+        if form.is_valid():
+            login(
+                request=request,
+                user=form.user_cache,
+            )
+            return redirect(
+                reverse_lazy("user_board", kwargs={"user_id": form.user_cache.id})
```

Comment:
There might be a `next` get parameter in the url.

With the current deployed version, if I try to access the url `https://backend.tilde.umuzi.org/users_and_teams_nav/` and I am not logged in then I get redirected to 
https://backend.tilde.umuzi.org/accounts/login/?next=/users_and_teams_nav/.

The idea is that when I log in I should get redirected back to where I wanted to be. It's good to have a default location in case `next` is empty, but it would be a better user experience if we pay attention to what the user was trying to do.